server/controllers/clients.js

```diff
@@ -0,0 +1,54 @@
+const Clients = require('../models/clients')
+
+const postClient = async (req,res)=>{
+    try {
+        const client = new Clients(req.body)
+        await client.save()
```

Comment:
not completely sure if we need to use await here, but i will take a look later, not a big deal